test/suite/test_import12.py

```diff
@@ -0,0 +1,208 @@
+
+        # Close the connection.
+        self.close_conn()
+
+        # Create a new database and connect to it.
+        newdir = 'IMPORT_DB'
+        # Construct the config string.
+        import_config = 'import=(enabled,repair=false,file_metadata=(' + \
+            original_db_file_config + '))'
+        import_config2 = 'import=(enabled,repair=false,panic_corrupt=false,file_metadata=(' + \
+            original_db_file_config + '))'
+        import_repair = 'import=(enabled,repair=true)'
+
+        for ck in range(0, self.max_ckpt + 1):
+            if ck % 2 == 0:
+                alter_config = self.alter_config2
+            else:
+                alter_config = self.alter_config1
+            self.pr("ck " + str(ck))
+            self.pr("remove and remake new database tree")
+            shutil.rmtree(newdir, ignore_errors=True)
+            os.mkdir(newdir)
+            self.pr("Open connection")
+            self.conn = self.setUpConnectionOpen(newdir)
+            self.pr("Open session")
+            self.session = self.setUpSessionOpen(self.conn)
+
+            # Make a bunch of files and fill them with data.
+            self.pr("populate and checkpoint")
+            self.populate(self.ntables, self.nrows)
+            self.session.checkpoint()
+
+            max_idx = orig_max_idx
+            min_idx = orig_min_idx
+            # Bring forward the oldest to be past or equal to the timestamps we'll be importing.
+            self.conn.set_timestamp('oldest_timestamp=' + self.timestamp_str(self.ts[max_idx]))
+
+            # Copy over the datafiles for the object we want to import.
+            copy_from = os.path.join('.', self.original_db_file)
+            copy_to = os.path.join(newdir, self.new_db_file)
+            shutil.copy(copy_from, copy_to)
+
+            # Do a dry-run of the import. Create and then drop.
+            # Import the file. Open and close a cursor because creating the table leaves it
+            # open and the drop returns EBUSY.
+            self.pr("Create import table " + self.new_uri)
+            self.session.create(self.new_uri, import_config)
+            self.session.checkpoint()
+            for i in range(0, ck):
+                # Perform a checkpoint. First one is a normal checkpoint.
+                # Any additional checkpoints use force.
+                self.pr("Checkpoint loop " + str(i))
+                self.session.checkpoint()
+                if i == 0:
+                    self.session.checkpoint()
+                else:
+                    # !!!
+                    # Doing a forced checkpoint as the second checkpoint causes the second
+                    # create/import below to fail because having two checkpoins renders the
+                    # original root page non-existent in the on-disk imported table that was
+                    # created-with-import and then dropped but leave the file on disk.
+                    self.session.checkpoint("force=true")
```

Comment:
Q: Just confirming that running this test with unfixed code shows the error?

Reply:
Yes, this was originally WT-13605 and it exactly reproduced the issue Atlas is seeing (whereas normal checkpoint does not because the table is clean).